Aptoide/aptoide-client-v8
# [AN-1641] Added multiple names support to the minimal card headers

What does this PR do?

   This pull-request adds the possibility to show multiple people that shared via the same social card and generated an Aggregated social card.
   This will show up multiple heads/names inside minimal cards of the Aggregated cards.

Where should the reviewer start?

- [ ] Aggregated Social install card
- [ ] Aggregated Social article card
- [ ] Aggregated Social video card
- [ ] Aggregated Social store latest card

How should this be manually tested?

  Open Aptoide v8 > Apps Timeline > Find aggregated cards > check if multiple people show up in some of the minimal cards. (if you are following ana2016, nzxt, trinkes15, it should happen, ask me about this).

What are the relevant tickets?

Tickets related to this pull-request: [AN-1641](https://aptoide.atlassian.net/browse/AN-1641).

Screenshots (if appropriate):
[Final result example](https://my.pcloud.com/publink/show?code=XZgYU3ZqlQ7G27pbqQd7jG5STRygVRA3KFk)

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
